### DISCUSSION TITLE
Usage of area outside of the app "circle" to hide the pop-up menu?

### DISCUSSION THREAD

**Discussion by misterperfectman** [rxhanson/Charmstone-Community#20] [Ideas]
Hi rxhanson!

First of all many thanks for the app.

Sometimes I launch Charmstone by accident and I want to cancel the pop-up menu. However, when I move my mouse far away from the pop-up menu, it does not disappear and still selects the app that is on the same horizontal/vertical pixel line as my mouse.

Could you implement something like an inverted safe area? As in, if you leave a specific area outside of the pop-up menu, it will cancel/hide the menu.

I am looking forward to hearing from you!

**Comment by rxhanson** [comment]
Thanks for the feedback. There's a slider for that, but it's kind of hidden. Select the far tab, and the slider is called "Outer Safe Area".

**Reply by misterperfectman** [reply]
Hi Ryan, thanks for your fast reply. I did not know, but I just used it and it works like a charm! Thank you!
